_posts/2021-12-20-knowledge-sharing-and-support.md

```diff
@@ -0,0 +1,22 @@
+
+It seems surprising, but many scientists still struggle daily, to share data, analyses and results conveniently with colleagues. Data may be too big to share in an email, which sadly remains a mainstay of scientific collaboration. Unstructured or proprietary formats like Word or Excel documents are poorly suited to unambiguously and reproducibly describing any specific analysis and results.
+
+Faithfully reproducing any specific computation is a requirement for the construction of scientific knowledge; enabling an efficient peer-reviewed research process must involve sharing a complete data analysis and reproducing computational environments (compute, storage), like the original one. In addition, it would allow rerun a few steps of analysis with slightly different parameters and other thresholds. However, this is almost never easily available, effectively excluding most researchers from participating in the essential scientific activities of analysis replication and of peer review. 
+
+Public infrastructure, like Wikipedia, can solve this problem by providing resources to store knowledge and a framework to harvest knowledge. A scientific framework of this kind should make it particularly easy to review the contributions of others and to access the collective body of analyses and experiments from which collective knowledge is derived.
+
+Galaxy provides many of the features needed for this kind of science infrastructure, and can already be used to overcome some of the challenges of routine reproducibility in data sciences. Want to avoid sharing results via email or USB keys? Easy! Use the framework feature that Galaxy provides to share a complete analysis history with your colleagues. The data is not copied, it's efficiently linked which lowers the global scientific costs. The original authors and any peers with access to it, can see every single tool and every single parameter of the shared analysis. Replication is built in, down to the level of being able to rerun any single step, using precisely the same software package versions as the original, even many years later. Jobs can be re-run, but with altered settings, to see how the results are affected. You can annotate every piece of your analysis with text and explanations to make it easier to understand and for your colleagues to harvest. The data never leaves the platform, so every user you have granted permissions to access the analysis, has the same computational resources to reproduce, or to improve the analysis. Use of this mechanism could help make review and publication processes more transparent and accessible to more eyeballs, leading to better science through improved communication of analyses and data.
```

Comment:
```suggestion
Public infrastructure, like Wikipedia, can solve this problem by providing resources to store knowledge and a framework to harvest knowledge. A scientific framework of this kind should make it particularly easy to review the contributions of others and to access the collective body of analyses and experiments from which collective knowledge is derived.

Galaxy provides many of the features needed for this kind of science infrastructure, and can already be used to overcome some of the challenges of routine reproducibility in data sciences. Want to avoid sharing results via email or USB keys? Easy! Use the framework feature that Galaxy provides to share a complete analysis history with your colleagues. The data is not copied but efficiently linked which lowers the global scientific costs. The original authors and any peers with access to it can see every single tool and every single parameter of the shared analysis. Replication is built-in, down to the level of being able to rerun any single step, using precisely the same software package versions as the original, even many years later. Jobs can be re-run, but with altered settings, to see how the results are affected. You can annotate every piece of your analysis with text and explanations to make it easier to understand and for your colleagues to harvest. The data never leaves the platform, so every user you have granted permissions to access the analysis, has the same computational resources to reproduce or to improve the analysis. To use of this mechanism could help make review and publication processes more transparent and accessible to more eyeballs, leading to better science through improved communication of analyses and data.
```